string arm
moveit_msgs/RobotTrajectory[] trajectory
int8[] control
string side
